python/paddle/optimizer/adamw.py

```diff
@@ -163,6 +165,9 @@ def __init__(self,
         self._apply_decay_param_fun = apply_decay_param_fun
         self._coeff = coeff
         self._lr_to_coeff = dict()
+        if lr_ratio is not None:
```

Comment:
should add explanation for the new lr_ration argument, which should follow the explanation for "apply_decay_param_fun"

Reply:
done.